### ISSUE TITLE
[Bug] 更新至v0.3.6后，无法访问CasaOS主页，docker容器正常运行

### ISSUE THREAD

**Issue by FlowerMagpie** [IceWhaleTech/CasaOS#606]
casaos@CasaOS:/etc$ ps aux
USER         PID %CPU %MEM    VSZ   RSS TTY      STAT START   TIME COMMAND
root           1  1.0  0.5  98368 10276 ?        Ss   09:26   0:02 /sbin/init
root           2  0.0  0.0      0     0 ?        S    09:26   0:00 [kthreadd]
root           3  0.0  0.0      0     0 ?        I<   09:26   0:00 [rcu_gp]
root           4  0.0  0.0      0     0 ?        I<   09:26   0:00 [rcu_par_gp]
root           5  0.0  0.0      0     0 ?        I    09:26   0:00 [kworker/0:0-events]
root           6  0.0  0.0      0     0 ?        I<   09:26   0:00 [kworker/0:0H-mmc_complete]
root           7  0.0  0.0      0     0 ?        I    09:26   0:00 [kworker/0:1-rcu_gp]
root           8  0.0  0.0      0     0 ?        I    09:26   0:00 [kworker/u8:0-ext4-rsv-conversion]
root           9  0.0  0.0      0     0 ?        I<   09:26   0:00 [mm_percpu_wq]
root          10  0.0  0.0      0     0 ?        S    09:26   0:00 [rcu_tasks_rude_]
root          11  0.0  0.0      0     0 ?        S    09:26   0:00 [rcu_tasks_trace]
root          12  0.0  0.0      0     0 ?        S    09:26   0:00 [ksoftirqd/0]
root          13  0.0  0.0      0     0 ?        I    09:26   0:00 [rcu_sched]
root          14  0.0  0.0      0     0 ?        S    09:26   0:00 [migration/0]
root          15  0.0  0.0      0     0 ?        S    09:26   0:00 [cpuhp/0]
root          16  0.0  0.0      0     0 ?        S    09:26   0:00 [cpuhp/1]
root          17  0.0  0.0      0     0 ?        S    09:26   0:00 [migration/1]
root          18  0.0  0.0      0     0 ?        S    09:26   0:00 [ksoftirqd/1]
root          19  0.0  0.0      0     0 ?        I    09:26   0:00 [kworker/1:0-events]
root          20  0.0  0.0      0     0 ?        I<   09:26   0:00 [kworker/1:0H-kblockd]
root          22  0.0  0.0      0     0 ?        I    09:26   0:00 [kworker/u8:1-flush-179:0]
root          23  0.0  0.0      0     0 ?        S    09:26   0:00 [kdevtmpfs]
root          24  0.0  0.0      0     0 ?        I<   09:26   0:00 [netns]
root          25  0.0  0.0      0     0 ?        S    09:26   0:00 [kauditd]
root          26  0.0  0.0      0     0 ?        S    09:26   0:00 [khungtaskd]
root          27  0.0  0.0      0     0 ?        S    09:26   0:00 [oom_reaper]
root          28  0.0  0.0      0     0 ?        I<   09:26   0:00 [writeback]
root          29  0.0  0.0      0     0 ?        S    09:26   0:00 [kcompactd0]
root          30  0.0  0.0      0     0 ?        SN   09:26   0:00 [ksmd]
root          31  0.0  0.0      0     0 ?        SN   09:26   0:00 [khugepaged]
root          49  0.0  0.0      0     0 ?        I<   09:26   0:00 [kintegrityd]
root          50  0.0  0.0      0     0 ?        I<   09:26   0:00 [kblockd]
root          51  0.0  0.0      0     0 ?        I<   09:26   0:00 [blkcg_punt_bio]
root          52  0.0  0.0      0     0 ?        I<   09:26   0:00 [edac-poller]
root          53  0.0  0.0      0     0 ?        I<   09:26   0:00 [devfreq_wq]
root          54  0.0  0.0      0     0 ?        I<   09:26   0:00 [kworker/0:1H-kblockd]
root          55  0.0  0.0      0     0 ?        I    09:26   0:00 [kworker/1:1-events]
root          56  0.0  0.0      0     0 ?        S    09:26   0:00 [kswapd0]
root          57  0.0  0.0      0     0 ?        I<   09:26   0:00 [kthrotld]
root          58  0.0  0.0      0     0 ?        I<   09:26   0:00 [acpi_thermal_pm]
root          59  0.0  0.0      0     0 ?        I    09:26   0:00 [kworker/0:2-cgroup_destroy]
root          60  0.0  0.0      0     0 ?        I    09:26   0:00 [kworker/0:3-events]
root          61  0.0  0.0      0     0 ?        I<   09:26   0:00 [ipv6_addrconf]
root          70  0.0  0.0      0     0 ?        I<   09:26   0:00 [kstrp]
root          73  0.0  0.0      0     0 ?        I<   09:26   0:00 [zswap-shrink]
root          74  0.0  0.0      0     0 ?        I<   09:26   0:00 [kworker/u9:0]
root         111  0.0  0.0      0     0 ?        I    09:26   0:00 [kworker/1:2-events]
root         113  0.0  0.0      0     0 ?        I<   09:26   0:00 [ata_sff]
root         114  0.0  0.0      0     0 ?        S    09:26   0:00 [scsi_eh_0]
root         115  0.0  0.0      0     0 ?        I<   09:26   0:00 [scsi_tmf_0]
root         117  0.0  0.0      0     0 ?        S    09:26   0:00 [scsi_eh_1]
root         119  0.0  0.0      0     0 ?        I<   09:26   0:00 [scsi_tmf_1]
root         121  0.0  0.0      0     0 ?        I    09:26   0:00 [kworker/u8:2-ext4-rsv-conversion]
root         122  0.0  0.0      0     0 ?        I    09:26   0:00 [kworker/u8:3-flush-179:0]
root         125  0.0  0.0      0     0 ?        I<   09:26   0:00 [sdhci]
root         126  0.0  0.0      0     0 ?        S    09:26   0:00 [irq/39-mmc0]
root         127  0.0  0.0      0     0 ?        I<   09:26   0:00 [kworker/1:1H-kblockd]
root         128  0.0  0.0      0     0 ?        I<   09:26   0:00 [mmc_complete]
root         129  0.0  0.0      0     0 ?        I<   09:26   0:00 [kworker/1:2H-kblockd]
root         130  0.1  0.0      0     0 ?        I<   09:26   0:00 [kworker/0:2H-kblockd]
root         131  0.0  0.0      0     0 ?        I<   09:26   0:00 [kworker/0:3H]
root         132  0.0  0.0      0     0 ?        I<   09:26   0:00 [kworker/0:4H-mmc_complete]
root         162  0.0  0.0      0     0 ?        S    09:26   0:00 [jbd2/mmcblk0p2-]
root         163  0.0  0.0      0     0 ?        I<   09:26   0:00 [ext4-rsv-conver]
root         201  0.1  1.1  48300 21384 ?        Ss   09:26   0:00 /lib/systemd/systemd-journald
root         221  0.1  0.3  22064  5708 ?        Ss   09:26   0:00 /lib/systemd/systemd-udevd
root         240  0.0  0.0      0     0 ?        S    09:26   0:00 [irq/127-mei_me]
root         249  0.0  0.0      0     0 ?        S    09:26   0:00 [watchdogd]
root         250  0.0  0.0      0     0 ?        I<   09:26   0:00 [kworker/1:3H-kblockd]
root         251  0.1  0.0      0     0 ?        I<   09:26   0:00 [kworker/1:4H-kblockd]
root         257  0.0  0.0      0     0 ?        I<   09:26   0:00 [cryptd]
root         283  0.0  0.0      0     0 ?        S    09:26   0:00 [card0-crtc0]
root         285  0.0  0.0      0     0 ?        S    09:26   0:00 [card0-crtc1]
root         288  0.0  0.0      0     0 ?        S    09:26   0:00 [card0-crtc2]
root         319  0.0  0.0      0     0 ?        I    09:26   0:00 [kworker/0:4-events]
systemd+     323  0.0  0.3  88400  5968 ?        Ssl  09:26   0:00 /lib/systemd/systemd-timesyncd
avahi        362  0.1  0.2   7276  4096 ?        Ss   09:26   0:00 avahi-daemon: running [CasaOS.local]
root         363  0.0  0.6 706224 11428 ?        Ssl  09:26   0:00 /usr/local/bin/casaos -c /casaOS/server/conf/conf.ini
root         364  0.0  0.1   6684  2668 ?        Ss   09:26   0:00 /usr/sbin/cron -f
message+     365  0.0  0.2   8532  4768 ?        Ss   09:26   0:00 /usr/bin/dbus-daemon --system --address=systemd: --nofork --nopidfile --systemd-activation --syslog-only
root         366  0.1  0.8 254520 16804 ?        Ssl  09:26   0:00 /usr/sbin/NetworkManager --no-daemon
root         368  0.0  0.3 233448  6704 ?        Ssl  09:26   0:00 /usr/libexec/polkitd --no-debug
root         369  0.0  0.2 220740  4080 ?        Ssl  09:26   0:00 /usr/sbin/rsyslogd -n -iNONE
root         370  0.0  0.2  11044  5168 ?        Ss   09:26   0:00 /usr/sbin/smartd -n
root         371  0.0  0.3  38520  7244 ?        Ss   09:26   0:00 /lib/systemd/systemd-logind
root         372  0.0  0.2  14560  5024 ?        Ss   09:26   0:00 /sbin/wpa_supplicant -u -s -O /run/wpa_supplicant
avahi        377  0.0  0.0   7092   344 ?        S    09:26   0:00 avahi-daemon: chroot helper
root         389  0.0  0.0      0     0 ?        I    09:26   0:00 [kworker/u8:4-ext4-rsv-conversion]
root         390  0.0  0.0      0     0 ?        I    09:26   0:00 [kworker/0:5-events_freezable]
root         391  0.0  0.0      0     0 ?        I    09:26   0:00 [kworker/0:6-events]
root         394  0.0  0.6 314784 12684 ?        Ssl  09:26   0:00 /usr/sbin/ModemManager
root         405  0.0  0.0   5784  1780 tty1     Ss+  09:26   0:00 /sbin/agetty -o -p -- \u --noclear tty1 linux
root         408  0.0  0.0      0     0 ?        I    09:26   0:00 [kworker/1:3-events]
root         414  0.2  2.8 1271124 53544 ?       Ssl  09:26   0:00 /usr/bin/containerd
root         419  0.0  0.4  13292  7880 ?        Ss   09:26   0:00 sshd: /usr/sbin/sshd -D [listener] 0 of 10-100 startups
root         445  0.3  4.5 1753020 85884 ?       Ssl  09:26   0:00 /usr/bin/dockerd -H fd:// --containerd=/run/containerd/containerd.sock
root         446  0.0  0.8  68400 16476 ?        Ss   09:26   0:00 /usr/sbin/nmbd --foreground --no-process-group
root         447  0.1  1.3  32004 24824 ?        Ss   09:26   0:00 python3 /usr/bin/wsdd --shortlog
root         457  0.0  1.3  82528 24744 ?        Ss   09:26   0:00 /usr/sbin/smbd --foreground --no-process-group
root         459  0.0  0.5  80380 10248 ?        S    09:26   0:00 /usr/sbin/smbd --foreground --no-process-group
root         460  0.0  0.3  80372  6196 ?        S    09:26   0:00 /usr/sbin/smbd --foreground --no-process-group
root         461  0.0  0.5  82528 10228 ?        S    09:26   0:00 /usr/sbin/smbd --foreground --no-process-group
root         467  0.0  0.0      0     0 ?        I    09:26   0:00 [kworker/u8:5-ext4-rsv-conversion]
root         468  0.0  0.0      0     0 ?        I    09:26   0:00 [kworker/u8:6-events_unbound]
root         635  0.0  0.2 1148908 5352 ?        Sl   09:26   0:00 /usr/bin/docker-proxy -proto tcp -host-ip 0.0.0.0 -host-port 22000 -container-ip 172.17.0.2 -container-port 22000
root         648  0.0  0.3 1076328 7436 ?        Sl   09:26   0:00 /usr/bin/docker-proxy -proto udp -host-ip 0.0.0.0 -host-port 22000 -container-ip 172.17.0.2 -container-port 22000
root         661  0.0  0.2 1222384 4696 ?        Sl   09:26   0:00 /usr/bin/docker-proxy -proto tcp -host-ip 0.0.0.0 -host-port 3000 -container-ip 172.17.0.3 -container-port 3000
root         671  0.0  0.3 1148652 6800 ?        Sl   09:26   0:00 /usr/bin/docker-proxy -proto tcp -host-ip :: -host-port 3000 -container-ip 172.17.0.3 -container-port 3000
root         686  0.0  0.4 1148652 8816 ?        Sl   09:26   0:00 /usr/bin/docker-proxy -proto udp -host-ip 0.0.0.0 -host-port 21027 -container-ip 172.17.0.2 -container-port 21027
root         698  0.0  0.2 1075176 4612 ?        Sl   09:26   0:00 /usr/bin/docker-proxy -proto tcp -host-ip 0.0.0.0 -host-port 8384 -container-ip 172.17.0.2 -container-port 8384
root         707  0.0  0.4 712856  9272 ?        Sl   09:26   0:00 /usr/bin/containerd-shim-runc-v2 -namespace moby -id b9cd68f29e72a8801fa00021ed44a203688e7341cb8aa481c7cd372e1471dce0 -address /run/contain
root         725  0.0  0.5 711704 10784 ?        Sl   09:26   0:00 /usr/bin/containerd-shim-runc-v2 -namespace moby -id e952bd61e41385030d3f06b34a295b02fc048a0c440115adffb66342c27300e7 -address /run/contain
root         745  0.0  0.5 712856 10744 ?        Sl   09:26   0:00 /usr/bin/containerd-shim-runc-v2 -namespace moby -id 586f83db331a65f68ed36c856d1f5a73440fccc762a7515f4f1269ce3447f476 -address /run/contain
root         770  0.0  0.4  14452  8792 ?        Ss   09:26   0:00 sshd: casaos [priv]
systemd+     783  0.0  0.2  16588  4772 ?        Ss   09:26   0:00 nginx: master process nginx -g daemon off;
root         793  0.0  0.0    204     4 ?        Ss   09:26   0:00 s6-svscan -t0 /var/run/s6/services
root         799  0.0  0.0    208    64 ?        Ss   09:26   0:00 /package/admin/s6/command/s6-svscan -d4 -- /run/service
root         950  0.0  0.0    212    64 ?        S    09:26   0:00 s6-supervise s6-linux-init-shutdownd
root         951  0.0  0.0    204     4 ?        Ss   09:26   0:00 /package/admin/s6-linux-init/command/s6-linux-init-shutdownd -c /run/s6/basedir -g 3000 -C -B
systemd+     954  0.0  0.1  17476  2136 ?        S    09:26   0:00 nginx: worker process
systemd+     955  0.0  0.1  17476  2136 ?        S    09:26   0:00 nginx: worker process
root         962  0.0  0.0    204     4 ?        S    09:26   0:00 s6-supervise s6-fdholderd
root        1025  0.0  0.0    212    64 ?        S    09:26   0:00 s6-supervise s6rc-oneshot-runner
root        1027  0.0  0.0    212    64 ?        S    09:26   0:00 s6-supervise s6rc-fdholder
root        1041  0.0  0.0    188     4 ?        Ss   09:26   0:00 /package/admin/s6/command/s6-ipcserverd -1 -- /package/admin/s6/command/s6-ipcserver-access -v0 -E -l0 -i data/rules -- /package/admin/s6/c
root        1274  0.0  0.0    204     4 ?        S    09:26   0:00 s6-supervise syncthing
casaos      1277  0.0  0.9 725196 16864 ?        Ssl  09:26   0:00 syncthing -home=/config -no-browser -no-restart --gui-address=0.0.0.0:8384
root        1299  0.0  0.0    212    60 ?        S    09:26   0:00 s6-supervise plex
casaos      1301  0.7  2.8 129724 53372 ?        Ssl  09:26   0:01 /usr/lib/plexmediaserver/Plex Media Server
casaos      1314  1.4  2.5 727308 46992 ?        SNl  09:26   0:02 syncthing -home=/config -no-browser -no-restart --gui-address=0.0.0.0:8384
casaos      1334  2.9  2.2  62376 41784 ?        SNl  09:26   0:05 Plex Plug-in [com.plexapp.system] /usr/lib/plexmediaserver/Resources/Plug-ins-819d3678c/Framework.bundle/Contents/Resources/Versions/2/Pyth
root        1339  0.0  0.4  14460  8672 ?        Ss   09:26   0:00 sshd: casaos [priv]
casaos      1342  0.0  0.4  15260  8596 ?        Ss   09:26   0:00 /lib/systemd/systemd --user
casaos      1343  0.0  0.1 101320  2696 ?        S    09:26   0:00 (sd-pam)
casaos      1367  0.0  0.3  14812  6644 ?        S    09:26   0:00 sshd: casaos@pts/0
casaos      1372  0.0  0.3  14660  6088 ?        S    09:26   0:00 sshd: casaos@notty
casaos      1374  0.0  0.2   8548  5260 pts/0    Ss   09:26   0:00 -bash
casaos      1377  0.0  0.2   5816  4324 ?        Ss   09:26   0:00 /usr/lib/openssh/sftp-server
casaos      1411  0.2  0.6  40168 11756 ?        Sl   09:26   0:00 /usr/lib/plexmediaserver/Plex Tuner Service /usr/lib/plexmediaserver/Resources/Tuner/Private /usr/lib/plexmediaserver/Resources/Tuner/Share
casaos      1458  2.0  2.2  61776 41628 ?        Sl   09:26   0:03 Plex Plug-in [com.plexapp.agents.imdb] /usr/lib/plexmediaserver/Resources/Plug-ins-819d3678c/Framework.bundle/Contents/Resources/Versions/2
root        1489  0.0  0.0      0     0 ?        I    09:27   0:00 [kworker/u8:7-ext4-rsv-conversion]
casaos      1513  0.0  0.1   9700  3416 pts/0    R+   09:30   0:00 ps aux

**Comment by LinkLeong** [comment]
What version before the upgrade?
Can you check the `systemctl status casaos-gateway.service`?
And what system and what device are you on?

**Comment by zhang-guanghao** [comment]
> What version before the upgrade? Can you check the `systemctl status casaos-gateway.service`? And what system and what device are you on?
I encountered the same problem. I installed cassos on a new environment. The linux distribution is debian. I checked the systemctl status casaos-gateway.service after installing. It shown "Unit casaos-gateway.service could not be found". How can I fix it.
![image](https://user-images.githubusercontent.com/7907975/197988438-5717217a-2366-4357-8ade-b0a9b1ed8513.png)

**Comment by zhang-guanghao** [comment]
> What version before the upgrade? Can you check the `systemctl status casaos-gateway.service`? And what system and what device are you on?
I encountered the same problem. I installed cassos on a new environment. The linux distribution is debian. I checked the systemctl status casaos-gateway.service after installing. It shown "Unit casaos-gateway.service could not be found". How can I fix it.
![image](https://user-images.githubusercontent.com/7907975/197988438-5717217a-2366-4357-8ade-b0a9b1ed8513.png)
